exampleExternalProviderScripts/chatExternalProviderScript.js

```diff
@@ -1,5 +1,4 @@
 this.web3inbox.chat.on('getMessages', ev => {
-  console.log('Received getMessage', ev, 'posting to', ev.id)
```

Comment:
@devceline Removed all logs, I think we shouldn't be shipped these to production. I still see many logs which is called from `pino` package. Should find a way to disable them I think